src/docs/sdk/features.mdx

```diff
@@ -396,6 +396,53 @@ The `fingerprints` field should be set to `["$operationName", "$operationType",
 }
 ```
 
+## GraphQL Performance
+
+The GraphQL Performance integration should match the guidelines for [GraphQL Client Integrations](/sdk/features/#graphql-client-integrations) with a few differences:
+
+The transaction's name should be set with the GraphQL operation name, if possible, otherwise fallback to something unique that makes sense, e.g. the canonical name of the actual/generated class.
+
+The transaction's description should be set with the GraphQL operation name, operation type and status code, if possible.
+
+The `request.api_target` should be set with `graphql`.
+
+The `request.data` should be set with the raw GraphQL request payload, if possible.
+
+The `contexts.response.data` should be set with the raw GraphQL response payload, only if there were `errors`.
+
+Spans should be created for [resolvers](https://www.apollographql.com/docs/apollo-server/data/resolvers/), if possible.
+
+Spans should be created for [data loaders](https://graphql.org/learn/best-practices/#server-side-batching-caching/), if possible.
+
+The operation type should follow the [Span Data Conventions](/sdk/performance/span-data-conventions/).
+
+Instrumenting APM for GraphQL will depend on the instrumented GraphQL library, if there are available hooks for it, the SDK should use them, otherwise, the SDK could try to monkeypatch the library or instrument the transport layer using heuristics, for example, if the URL ends with `graphql`, if there are HTTP Headers, etc.
+
+If there are hooks available and the transport layer is also instrumented (e.g. Apollo Interceptors for GraphQL and Spring), the SDK should give preference to the layer that has more information and void creating duplicate transactions/spans, or merge the information, if possible.
+
+Changes in the product may be necessary, e.g. if `request.api_target` is set to `graphql`, the `request.data` and `contexts.response.data` should do syntax highlighting.
```

Comment:
This is probably a TODO, @malwilley did the implementation for errors, we could reuse the code for performance.